client/js/components/statement/assessmentTable/DpFilterModal.vue

```diff
@@ -187,7 +188,7 @@
         </div>
 
         <!-- Button row -->
-        <div class="text--right space-inline-s">
+        <div class="text-right space-inline-s">
```

Comment:
i think this should also be
```suggestion
        <div class="text--right...
```
like in :79

Reply:
It is `text-right`, as the respective utility classes [have been moved to tailwind](https://github.com/demos-europe/demosplan-core/pull/1544).